### DISCUSSION TITLE
Generate types for custom schema?

### DISCUSSION THREAD

**Discussion by jeyjeycodes** [supabase/supabase#5449] [Questions]
Hi all!

Is it possible to generate types for a custom schema that isn't public?
I don't use the public schema at all, this would be amazing!

Kind regards,
another developer breaking stuff

**Comment by 5hee75** [comment]
I'll leave a note here to boost visibility - but I am also wondering how to do this.  I use `public` for most things, but I also use my own schemas and would like the types generated for the `auth` and `storage` schemas, as well.

**Comment by magick93** [comment]
This feature has just been released - see https://github.com/supabase/cli/issues/508#issuecomment-1277090494
